kubernetes/test-infra/kops-on-aws/main.tf

```diff
@@ -35,15 +35,27 @@ data "http" "ipinfo" {
 }
 
 resource "tls_private_key" "ssh" {
-  algorithm   = "RSA"
+  algorithm = "RSA"
+}
+
+resource "aws_s3_bucket" "kops_state_store" {
+  bucket = "${local.bucket_name}"
+  acl    = "private"
+
```

Comment:
Are you sure we don't need [`force_destroy = true`](https://www.terraform.io/docs/providers/aws/r/s3_bucket.html#force_destroy) here? I remember kops can sometimes leave files in the bucket even after destroying the cluster.

Reply:
That's a good point. It won't clean it if there are files still in there. I'll add a `force_destroy`.